backend/src/v5/schemas/tickets/templates.constants.js

```diff
@@ -61,15 +74,27 @@ const viewGroups = createConstantMapping([
 const riskLevelsArr = ['Very Low', 'Low', 'Moderate', 'High', 'Very High'];
 const riskLevels = createConstantMapping(riskLevelsArr);
 
-const createPropertyEntry = (name, type, values, defaultVal, readOnly, availableIf) => deleteIfUndefined({
-	name, type, values, default: defaultVal, readOnly, availableIf });
+const createPropertyEntry = (name, type, config = {}) => {
+	const unrecognisedOptions = getArrayDifference(Object.values(propOptions), Object.keys(config));
+
+	/* istanbul ignore next */
+	if (unrecognisedOptions.length) {
+		// we don't technically need this check - hence no test case, it's here to make sure we know if we've made a typo
+		throw new Error(`Unrecognised configuration: ${unrecognisedOptions.join(',')}, provided ${Object.keys(config).join(',')}`);
+	}
+	return deleteIfUndefined({
+		name, type, ...config });
+};
 
 const TemplateConstants = { propTypes, presetEnumValues, presetModules, riskLevels, viewGroups };
 
 TemplateConstants.riskLevelsToNum = (value) => riskLevelsArr.indexOf(value);
 
 TemplateConstants.statusTypes = ['open', 'active', 'review', 'done', 'void'];
 
+const riskLevelConfig = { [propOptions.VALUES]: riskLevelsArr, [propOptions.DEFAULT]: riskLevels.VERY_LOW };
+const clashElementTypes = ['Revit, IFC, DWG, DGN, Unknown'];
```

Comment:
this is an array of 1 string instead of 5

Reply:
:o good spot 😆